excella-core/excella-trans
# Improve unit tests

- Make WorkbookTest abstract
- Remove throws/not throws expected `fail()` usage.
- Migrate from JUnit 4 to JUnit 5
- Remove primitive wrapper class's constructor usage in test class
